src/Config/Config.php

```diff
@@ -61,7 +61,9 @@ public function addPath($path)
         // Ensure paths are loaded
         $this->getPaths();
 
-        $this->paths[] = $path;
+        if (!in_array($path, $this->paths, true)) {
+            $this->paths[] = $path;
+        }
```

Comment:
What do you think about solving this with a `array_unique()` call instead? (That would also nicely clean up already-"corrupted" JSON files.)